web/app/AttributeFormatter.js

```diff
@@ -74,6 +74,18 @@ Ext.define('Traccar.AttributeFormatter', {
         }
     },
 
+    geofenceIdFormatter: function (value) {
+        var geofence, store;
+        if (value !== 0) {
+            store = Ext.getStore('AllGeofences');
```

Comment:
Yes, we spent much time in optimizing it https://github.com/tananaev/traccar-web/pull/507